x/group/server/invariants.go

```diff
@@ -0,0 +1,97 @@
+package server
+
+import (
+	"fmt"
+	"math"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/require"
+
+	"github.com/regen-network/regen-ledger/orm"
+	"github.com/regen-network/regen-ledger/x/group"
+)
+
+type Invar struct {
+	sdkCtx        sdk.Context
+	proposalTable orm.AutoUInt64Table
+}
+
+func RegisterInvariants(ir sdk.InvariantRegistry, invar Invar) {
+	ir.RegisterRoute(group.ModuleName, "Tally-Votes", tallyVotesInvariant(invar))
+}
+
+func AllInvariants(invar Invar) sdk.Invariant {
+	return func(ctx sdk.Context) (string, bool) {
+		return tallyVotesInvariant(invar)(ctx)
+	}
+}
+
+func tallyVotesInvariant(invar Invar) sdk.Invariant {
```

Comment:
keep having some `func (s serverImpl) tallyVotesInvariant() sdk.Invariant`but then call some private function like this one in it

Reply:
Yeah, sure